bitnami/jenkins/templates/deployment.yaml

```diff
@@ -532,15 +536,15 @@ spec:
             sources:
               {{- if .Values.initScripts }}
               - configMap:
-                  name: {{ template "jenkins.initScripts" . }}
+                  name: {{ template "common.names.fullname" . }}-init-scripts
```

Comment:
Let's define the `jenkins.initScripts` in the [_helpers.tpl](https://github.com/bitnami/charts/blob/main/bitnami/jenkins/templates/_helpers.tpl) instead:
```
{{/*
Get the initialization scripts ConfigMap name.
*/}}
{{- define "jenkins.initdbScriptsCM" -}}
{{- if .Values.initScriptsCM -}}
  {{- printf "%s" (tpl .Values.initScriptsCM $) -}}
{{- else -}}
  {{- printf "%s-init-scripts" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" -}}
{{- end -}}
{{- end -}}
```
Then, you can simplify this like:
```yaml
              {{- if or .Values.initScripts .Values.initScriptsCM }}
              - configMap:
                  name: {{ template "jenkins.initdbScriptsCM" . }}
              {{- end }}
```

Reply:
@juan131  I accept this review.